src/pages/CropSidebar/Filters/Filters.js

```diff
@@ -19,19 +19,19 @@ const DollarsAndRatings = ({ filter, handleChange }) => {
 
   return (
     <div style={style}>
-      {new Array(filter.values.length)
+      {new Array(5)
```

Comment:
they aren't all 5 though right? this will break the currency ratings i think

Reply:
it's not showing the dollar signs anymore but it didn't do what i expected and have 5 buttons to click